README.md

```diff
@@ -117,4 +117,214 @@ containers:
+      connectionStringsStandardSrv: mongodb+srv://test.iojsa.mongodb.net
+      instanceSizeName: M10
+      projectID: 6065e15b16c0731bf3a2aefa
+      projectName: Project 2
+      providerName: AWS
+      regionName: US_EAST_1
+    name: test
+  - instanceID: 606b2ffbc9a90e310e642482
+    instanceInfo:
+      connectionStringsStandard: ""
+      connectionStringsStandardSrv: mongodb+srv://dbcreatedinatalas.iojsa.mongodb.net
+      instanceSizeName: M0
+      projectID: 6065e15b16c0731bf3a2aefa
+      projectName: Project 2
+      providerName: TENANT
+      regionName: US_EAST_1
+    name: DBCreatedInAtalas
+  - instanceID: 60b7a72f4877d05880c487d2
+    instanceInfo:
+      connectionStringsStandard: mongodb://test-shard-00-00.fbbz5.mongodb.net:27017,test-shard-00-01.fbbz5.mongodb.net:27017,test-shard-00-02.fbbz5.mongodb.net:27017/?ssl=true&authSource=admin&replicaSet=atlas-eituqf-shard-0
+      connectionStringsStandardSrv: mongodb+srv://test.fbbz5.mongodb.net
+      instanceSizeName: M10
+      projectID: 60b798fea37f9f09acc1a154
+      projectName: mytest
+      providerName: AWS
+      regionName: US_EAST_1
+    name: test
+```
+**3.** Check MongoDBAtlasConnection Custom Resource
+
+The application developer can then create a [DBaaSConnection](https://github.com/RHEcosystemAppEng/dbaas-operator/blob/main/config/crd/bases/dbaas.redhat.com_dbaasconnections.yaml) CR for connection to the MongoDB database instace found, the DBaaS Operator automatically creates a MongoDBAtlasConnection CR. The Atlas Operator creates a database user in Atlas for the cluster with the default database `admin`. The Atlas Operator stores the db user corredentials in a kubernetes secret, and the remaining connection information in a configmap, and then updates the MongoDBAtlasConnection CR status.
+
+Here is an example of MongoDBAtlasConnection CR.
+```
+apiVersion: dbaas.redhat.com/v1alpha1
+kind: MongoDBAtlasConnection
+metadata:
+  name: test-dbaas-connection
+  namespace: sgahlot-test-op
+  ownerReferences:
+  - apiVersion: dbaas.redhat.com/v1alpha1
+    blockOwnerDeletion: true
+    controller: true
+    kind: DBaaSConnection
+    name: test-dbaas-connection
+    uid: 77193619-6ab1-43c9-acf2-a40c2cfe7703
+  resourceVersion: "950339068"
+  selfLink: /apis/dbaas.redhat.com/v1alpha1/namespaces/sgahlot-test-op/mongodbatlasconnections/test-dbaas-connection
+  uid: a50b06db-8fa1-45c9-9893-833a028dfccc
+spec:
+  instanceID: 606b2ffbc9a90e310e642482
+  inventoryRef:
+    name: dbaas-mytest
+    namespace: openshift-operators
+status:
+  conditions:
+  - lastTransitionTime: "2021-08-18T20:07:51Z"
+    message: ""
+    reason: Ready
+    status: "True"
+    type: ReadyForBinding
+  connectionInfoRef:
+    name: atlas-connection-cm-knp9z
+  credentialsRef:
+    name: atlas-db-user-5pc8b
+```
+The corresponding generated secret:
+```
+apiVersion: v1
+data:
+  password: cGFzczEyM3dvcmRA
+  username: ZGJVc2VyXzEwMQ==
```

Comment:
please redact any live URLs or credentials - these are an easy decode away from giving the world a target URL & credentials to access it!

Reply:
Done!